tasks/release.js

```diff
@@ -51,6 +51,15 @@ module.exports = function (grunt) {
         platform.debName && platform.debName + '.sha1.txt'
       );
     }, [])
+    .reduce((files, file) => {
```

Comment:
I believe we'll get this for free if we add windows-x64 and windows-x86 to https://github.com/epixa/kibana/blob/7ac1efe859d865e4159fd42705e8856f4035470b/tasks/config/platforms.js#L13